src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -70,10 +70,10 @@ public static void validateNumOfArgs(Object[] args, int min, int max)
 
         if (args.length < min || args.length > max) {
             throw new ParseException("Invalid number of arguments!"
-                    + "Number of arguments should be more than or equal to "
-                    + min
-                    + " and less than or equal to "
-                    + max);
+                    + " Number of arguments should be more than or equal to " + min
+                    + " and less than or equal to " + max
+                    + "\nInput oder: [CODE], [YEAR], [SEMESTER], [CREDITS], [GRADE]"
```

Comment:
The example kind of implies that all args are necessary when this is not usually the case, particularly when we have `edit`/`update`/`delete` etc. Additionally `validateNumArgs` seems to be a general purpose utility function that could be used even for non-module commands such as `set cap goal`. 

To make it more flexible, perhaps this function should take in a `inputSuggestion` string that is supplied by the specific command parser. 

Also, typo in `oder`.